test/reproducibility_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+bazel clean
+bazel build test:play-routes
+routes_hash0=$(md5sum bazel-bin/test/play/routes/play_routes/router/Routes.scala | awk '{print $1}')
+prefix_hash0=$(md5sum bazel-bin/test/play/routes/play_routes/router/RoutesPrefix.scala | awk '{print $1}')
+
+sleep 2
+bazel clean
+bazel build test:play-routes
+routes_hash1=$(md5sum bazel-bin/test/play/routes/play_routes/router/Routes.scala | awk '{print $1}')
```

Comment:
Because this is re-used, we might want to put it into a bash function rather than just repeat the code.